doc/porting/cmsis_rtos_v1.rst

```diff
@@ -0,0 +1,11 @@
+.. _cmsis_rtos_v1:
+
+CMSIS RTOS v1
+##########################
+
+CMSIS expands into Cortex-M Software Interface Standard. CMSIS RTOS is a
```

Comment:
A bit clearer would be to start this paragraph with:

    Cortex-M Software Interface Standard (CMSIS) RTOS is a
    vendor-independent ...

Reply:
will make the change. Thanks David.